maintenance/static/css/main.css

```diff
@@ -7473,38 +7473,71 @@ a.text-gray-dark:focus, a.text-gray-dark:hover {
 .twitter-glyph:hover::before, .twitter-glyph:focus::before {
```

Comment:
Thanks for taking the maintenance page into consideration. What we have currently on `maintenance/static/css/main.css` is more than what the maintenance page actually needs. Since we don't show any form element on that page, we don't need to worry about updating `maintenance/static/css/main.css`. 

<img width="1250" alt="image" src="https://github.com/user-attachments/assets/e544356a-f11a-4fc5-8ca9-dad87af6cb1d">

Reply:
Yeah you're right, I'll discard those changes. Thanks Mavis.